src/components/message-input/index.tsx

```diff
@@ -42,6 +44,7 @@ interface State {
   isEmojisActive: boolean;
   isGiphyActive: boolean;
   isMicActive: boolean;
+  isMessengerFullScreen?: boolean;
```

Comment:
why do we have this in the State AND in the Properties?

Reply:
Good spot. Removed.